.editorconfig

```diff
@@ -27,3 +27,7 @@ function_next_line = false
 
 [libs/**]
 ignore = true
+
+[*.py]
+# This is the largest PEP-8 compliant(ish - long story)
```

Comment:
Sorry for being unfunny, but the long story part should be either explained or dropped.

Reply:
No problem, updated